draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -168,68 +181,37 @@ including obvious identifiers like HTTP cookies across the requests.
 ~~~
 {: #diagram-dualconnect title="Diagram of making separate connections to generate separate contexts"}
 
-The privacy-oriented protocols described in this document generally involve more complex
-partitioning, but the techniques to partition communication contexts still employ the
+## Context Separation
+
+In order to define an analyze how various partitioning techniques work, the boundaries of what is
+being partitioned need to be established. This is the role of context separation. In particular,
+in order to prevent correlation of user-specific information across contexts, partitions need
+to ensure that any single entity (other than the client itself) does not participate in contexts
+where both identities are visible.
+
+Context separation can be achieved in different ways, e.g. over time, across network paths, based
+on (en)coding, etc. The privacy-oriented protocols described in this document generally involve
+more complex partitioning, but the techniques to partition communication contexts still employ the
 same techniques:
 
 1. Encryption allows partitioning of contexts within a given network path.
+1. Using separate connections across time or space allow partitioning of contexts for different
+application transactions.
 
-1. Using separate connections across time and/or space allow partitioning of contexts for
-different transactions.
-
-## Identities and Correlation
-
-Creating separate communication contexts is one important aspect of partioning, but
-ensuring that the contexts are split to prevent correlation of identities is necessary
-to make the partitions effective for improving privacy.
-
-{{?RFC6973}} discusses the importance of identifiers in reducing correlation:
-
-"Correlation is the combination of various pieces of information related to an individual
-or that obtain that characteristic when combined... Correlation is closely related to
-identification.  Internet protocols can facilitate correlation by allowing individuals'
-activities to be tracked and combined over time."
-
-{{?RFC6973}} goes on to discuss different ways that identities can be obscured: anonymity
-and pseudonymity.
-
-"To enable anonymity of an individual, there must exist a set of individuals that appear to
-have the same attribute(s) as the individual."
-
-"Pseudonymity is strengthened when less personal data can be linked to the pseudonym; when
-the same pseudonym is used less often and across fewer contexts; and when independently
-chosen pseudonyms are more frequently used for new actions (making them, from an observer's or
-attacker's perspective, unlinkable)."
-
-The "anonymity level" of a given identity exists on a scale, not a clear line between real
```

Comment:
I think it's still useful to have a subsection that mentions identity as seen within a context?

Reply:
I don't know if it's useful to zero in on identity since it's just one type of user-specific information. I think it's more helpful to just say that "each context has at most one type of user-specific information."

Reply:
(Discussed offline) Pushed back some text from RFC6973.